### PR TITLE
[API] Refactor images path resolving

### PR DESCRIPTION
**Pull request by NoResponseMate**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | 1.13 |
| Bug fix?        | kinda                                                       |
| New feature?    | kinda                                                       |
| BC breaks?      | no?                                                       |
| Deprecations?   | no |
| Related tickets | -                      |
| License         | MIT                                                          |

### CODE REVIEW

File: src/Sylius/Bundle/ApiBundle/Resources/config/services/providers.xml [hunk 1]
```diff
@@ -23,7 +23,7 @@
             <argument>%sylius.security.new_api_route%</argument>
         </service>
 
-        <service id="Sylius\Bundle\ApiBundle\Provider\ProductImageFilterProviderInterface" class="Sylius\Bundle\ApiBundle\Provider\LiipProductImageFilterProvider">
+        <service id="Sylius\Bundle\ApiBundle\Provider\ImageFiltersProviderInterface" class="Sylius\Bundle\ApiBundle\Provider\LiipImageFiltersProvider">
             <argument>%liip_imagine.filter_sets%</argument>
         </service>
```
Comment (Wojdylak) [inline src/Sylius/Bundle/ApiBundle/Resources/config/services/providers.xml]:
```suggestion
        <service id="sylius.api.provider.lipp_image_filters" class="Sylius\Bundle\ApiBundle\Provider\LiipImageFiltersProvider">
            <argument>%liip_imagine.filter_sets%</argument>
        </service>
        <service id="Sylius\Bundle\ApiBundle\Provider\ImageFiltersProviderInterface" alias="sylius.api.provider.lipp_image_filters">
```

**Review by Wojdylak — DISMISSED** [review]

**Review by Wojdylak — APPROVED** [review]

File: src/Sylius/Bundle/ApiBundle/DependencyInjection/Configuration.php [hunk 2]
```diff
@@ -54,8 +54,8 @@ public function getConfigTreeBuilder(): TreeBuilder
                 ->end()
             ->end()
             ->children()
-                ->variableNode('product_image_prefix')
-                    ->defaultValue('media/image')
```
Comment (jakubtobiasz) [inline src/Sylius/Bundle/ApiBundle/DependencyInjection/Configuration.php:58]: Why did you remove this parameter? It seems unrelated, and there's no single word about it in the upgrade file.

**Review by jakubtobiasz — APPROVED** [review]

**Comment by jakubtobiasz** [comment]
Thank you, @NoResponseMate!

File: src/Sylius/Bundle/ApiBundle/DependencyInjection/Configuration.php (hunk 2 above)
Comment (NoResponseMate) [reply src/Sylius/Bundle/ApiBundle/DependencyInjection/Configuration.php:58]: Added in #16037
